components/API/DatasetsList/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './style.scss';
+
+class DatasetsList extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      datasets: []
+    };
+  }
+
+  render() {
+    const { applications } = this.props;
+    let { url } = this.props;
+
+    this.getDatasets(applications.join(','));
```

Comment:
Call this in the WillMount instead of the render function. You can also make the join inside the function